src/components/EditPartnership/EditPartnership.tsx

```diff
@@ -10,11 +24,89 @@ interface EditPartnershipProps {
 
 export const EditPartnership: FC<EditPartnershipProps> = ({
   state,
-  // partnership,
+  partnership,
 }) => {
+  const [updatePartnership] = useUpdatePartnershipMutation();
+  const [deletePartnership] = useDeletePartnershipMutation();
+  const { enqueueSnackbar } = useSnackbar();
+
+  const submit = async (input: UpdatePartnershipInput) => {
+    const inputWithId: UpdatePartnershipInput = {
+      partnership: { ...input.partnership, id: partnership.id },
+    };
+    await updatePartnership({
+      variables: { input: inputWithId },
+      refetchQueries: ['ProjectPartnerships'],
+    });
+    enqueueSnackbar(
+      `Updated Partnership with ${partnership.organization.name.value}`,
+      {
+        variant: 'success',
+      }
+    );
+  };
+
+  const onDeleteClick = async () => {
+    const { id } = partnership;
+    await deletePartnership({
+      variables: { input: id },
+      refetchQueries: ['ProjectPartnerships'],
+    });
+    state.onClose();
+    enqueueSnackbar(
+      `Deleted Partnership with ${partnership.organization.name.value}`,
+      {
+        variant: 'success',
+      }
+    );
+  };
+
+  // TODO: have DialogForm pass down initialValues to the radio options (currently passed to Checkboxes)
+  // const initialValues = {
+  //   partnership: {
+  //     id: partnership.id,
+  //     agreementStatus: partnership.agreementStatus.value,
+  //     mouStatus: partnership.mouStatus.value,
+  //     types: partnership.types.value,
+  //   },
+  // };
+
   return (
-    <DialogForm onSubmit={() => null} {...state}>
-      Edit Partnerships Component
+    <DialogForm
+      onSubmit={submit}
+      {...state}
+      DialogProps={{
+        fullWidth: true,
+        maxWidth: 'xs',
+      }}
+      title="Edit Partnership"
+      leftAction={<Button onClick={onDeleteClick}>Delete</Button>}
+      // initialValues={initialValues}
+    >
+      <SubmitError />
+      <RadioField name="partnership.agreementStatus" label="Agreement Status">
+        <RadioOption label="Not Attached" value="NotAttached" />
+        <RadioOption label="Awaiting Signature" value="AwaitingSignature" />
+        <RadioOption label="Signed" value="Signed" />
+      </RadioField>
+      <RadioField name="partnership.mouStatus" label="Mou Status">
+        <RadioOption label="Not Attached" value="NotAttached" />
+        <RadioOption label="Awaiting Signature" value="AwaitingSignature" />
+        <RadioOption label="Signed" value="Signed" />
+      </RadioField>
+      <CheckboxesField name="partnership.types" label="Partnership Type" row>
```

Comment:
```suggestion
      <CheckboxesField name="partnership.types" label="Partnership Types" row>
```
or maybe even just "Types".

Can you also move this to the top? It's the most important field.

Reply:
Moved types up and disabled if `canEdit: false` although it doesn't work as expected on the radio fields.  The radio fields are still able to be toggled